components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApiProductsApiServiceImpl.java

```diff
@@ -948,12 +948,6 @@ private APIProductDTO getAPIProductByID(String apiProductId, APIProvider apiProv
         try {
             String tenantDomain = RestApiCommonUtil.getLoggedInUserTenantDomain();
             APIProduct api = apiProvider.getAPIProductbyUUID(apiProductId, tenantDomain);
-            APIRevision apiRevision = ApiMgtDAO.getInstance().checkAPIUUIDIsARevisionUUID(apiProductId);
```

Comment:
I think we need these entries for UI purposes, to differentiate Current API Product and Revision API Product

Reply:
Accessing dao from rest api level was not a good practise hence move into apiproviderimpl